api/sql/schema/750$transfer.transfer.get.sql

```diff
@@ -92,6 +93,8 @@ LEFT JOIN
     [transfer].[partner] pl ON pl.partnerId = t.ledgerId
 LEFT JOIN
     [transfer].[event] e ON e.transferId = t.transferId AND e.source = 'acquirer' AND e.type = 'transfer.push'
+LEFT JOIN
+    [transfer].[event] er ON er.transferId = t.transferId AND er.source = 'acquirer' AND er.state <> 'request'
```

Comment:
The join condition does not look very good, as I think it can return more than one row. The condition should be more precise to ensure only one match.